delta/plugins/elasticsearch/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/query/DefaultViewSearchSuite.scala

```diff
@@ -155,27 +159,34 @@ class DefaultViewSearchSuite
-  private val excludeDatasetType        = ResourcesSearchParams(types = List(ExcludedType(nxv + "Dataset")))
-  private val byDeprecated              = ResourcesSearchParams(deprecated = Some(true))
-  private val byCreated                 = ResourcesSearchParams(createdBy = Some(alice))
-  private val between_8_and_16          = TimeRange.Between.unsafe(epochPlus(8L), epochPlus(16))
-  private val byCreatedBetween_8_and_16 = ResourcesSearchParams(createdAt = between_8_and_16)
-  private val byCreatedAfter_11         = ResourcesSearchParams(createdAt = TimeRange.After(epochPlus(11L)))
-  private val byUpdated                 = ResourcesSearchParams(updatedBy = Some(alice))
-  private val byUpdated_Before_12       = ResourcesSearchParams(updatedAt = TimeRange.Before(epochPlus(12L)))
+  private val all                                        = ResourcesSearchParams()
+  private val orgByType                                  = ResourcesSearchParams(types = List(IncludedType(orgType)))
+  private val datasetAndCellTypes: ResourcesSearchParams =
+    ResourcesSearchParams(types = List(IncludedType(datasetType), IncludedType(cellType)), typeOperator = And)
+  private val datasetOrCellTypes: ResourcesSearchParams  =
+    ResourcesSearchParams(types = List(IncludedType(datasetType), IncludedType(cellType)))
+  private val orgBySchema                                = ResourcesSearchParams(schema = Some(orgSchema))
+  private val excludeDatasetType                         = ResourcesSearchParams(types = List(ExcludedType(datasetType)))
+  private val byDeprecated                               = ResourcesSearchParams(deprecated = Some(true))
+  private val byCreated                                  = ResourcesSearchParams(createdBy = Some(alice))
+  private val between_8_and_16                           = TimeRange.Between.unsafe(epochPlus(8L), epochPlus(16))
+  private val byCreatedBetween_8_and_16                  = ResourcesSearchParams(createdAt = between_8_and_16)
+  private val byCreatedAfter_11                          = ResourcesSearchParams(createdAt = TimeRange.After(epochPlus(11L)))
+  private val byUpdated                                  = ResourcesSearchParams(updatedBy = Some(alice))
+  private val byUpdated_Before_12                        = ResourcesSearchParams(updatedAt = TimeRange.Before(epochPlus(12L)))
 
   private val bbpResource    = bbp.asResourceF
   private val byId           = ResourcesSearchParams(id = Some(bbpResource.id))
   private val byLocatingId   = ResourcesSearchParams(locate = Some(bbpResource.id))
   private val byLocatingSelf = ResourcesSearchParams(locate = Some(bbpResource.self))
 
   // Action / params / matching resources
+
   List(
     ("all resources", all, allResources),
     ("org resources by type", orgByType, orgs),
+    ("types AND", datasetAndCellTypes, List(cell)),
+    ("types OR", datasetOrCellTypes, List(trace, cell)),
```

Comment:
Maybe some test for the excluded ones ?

Reply:
This can't be done because it doesn't work, opening a bug and ignoring for now